src/UI/Pages/ResourceDetails/Tabs/LogTab/Details.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ComponentProps } from "react";
+import { Story } from "@storybook/react/types-6-0";
+import { LogLevelString } from "@/Core";
+import { Details } from "./Details";
+
+export default {
+  title: "ResourceLog",
+  component: Details,
+};
+
+const Template: Story<ComponentProps<typeof Details>> = (args) => (
```

Comment:
I like to avoid this Template syntax by storybook as you no longer have typescript safety and after a while you have to fix all stories once you notice it.